source/_data/glossary.yml

```diff
@@ -292,6 +292,16 @@
 
   link: /docs/scripts/service-calls/
 
+- term: State
+  definition: |-
+    The state holds the information of interest of an entity. E.g. if a light is on or off, the current temperature, or the amount of energy used. The data type of state is `string` (a textual value). Entities store 2 timestamps related to the state: `last_updated` and `last_changed`. Each entity has exactly one state and the state only holds one value at a time. However, entities can store attributes related to that state. E.g. the state of a light is _on_, and the related state attributes could be its current brightness and color values.
+
+    State changes can be used as the source of triggers. The current state can be used in conditions.
+    
+  link: /docs/scripts/configuration/state_object/
+  excerpt: >
+    State holds the information of interest of an entity. E.g. if a light is on or off. Each entity has exactly one state and state only holds one value at a time. However, entities can store attributes related to that state. E.g. friendly name, unit of measurement, etc.
```

Comment:
Small tweaks

```suggestion
    The state holds the information of interest of an entity. E.g. if a light is on or off. Each entity has exactly one state and the state only holds one value at a time. However, entities can store attributes related to that state. E.g. brightness, color, a unit of measurement, etc.
```

Reply:
Thank you for the catch! Included these changes in the new commit to replace the _e.g._